src/layouts/DashboardLayout.js

```diff
@@ -1,62 +1,81 @@
 import React from 'react';
-import { Layout, Menu, Icon } from 'antd';
+import { Layout, Menu, Icon, Dropdown } from 'antd';
 import { Link } from 'react-router-dom';
 import { injectIntl } from 'react-intl'; 
 import AppFooter from './AppFooter';
 import logo from '../assets/logo.png'
 import './DashboardLayout.less'
+import { UserContext } from '../UserContext';
 
 const { Content, Sider } = Layout;
 
+const onClick = ({ key }) => {
+  localStorage.removeItem('ws-token');
+  sessionStorage.removeItem('ws-token');
+  window.location.href="/login"
```

Comment:
Do not jump like this, this will cause the whole page to actually reload instead of the "reload" in react.
To jump properly: https://stackoverflow.com/questions/31079081/programmatically-navigate-using-react-router
Maybe you don't even need to jump manually, just toggle the user context refresh (https://github.com/websentry/WebSentry-FrontEnd/blob/11891474850ddb6fc516d4d3c10ac740afb1a4ce/src/UserContext.js#L8), I think the dashboard layout will notice the user is not logged in anymore and jump to login page.